osu.Game.Rulesets.Osu/Difficulty/Skills/Aim.cs

```diff
@@ -50,7 +50,7 @@ private double strainValueOf(DifficultyHitObject current)
             if (osuLastObj.BaseObject is Slider && withSliders)
```

Comment:
Confirm that this is still correct as `osuLastObj` here? Looks weird with the changes inside this conditional now reading from one index back while the conditional itself isn't.

Reply:
This is one of the reasons for me changing it, because it was using `osuCurrObj.TravelDistance` to access the travel distance of `osuLastObj`.

So yes, this condition is still fine as it is, the only change is that the properties apply directly to the relevant hitobject now.

Reply:
To add on as to why - all these blocks of code try to do (perhaps not explained enough in the leading comments?) is to calculate the velocity through a previous slider into the current object. One part is stored in the previous object (travel), and the other is stored in the current object (jump / movement, however you want to think about it).